deploy/clowdapp.yaml

```diff
@@ -119,7 +119,17 @@ objects:
                         number: 24816
       EOF
 
-      kubectl patch pulp cs-pulp --type='json' -p='[{"op": "replace", "path": "/spec/pulp_settings/content_origin", "value": "'https://"${HOST}"'"}]'
+      # Sometimes the content_origin is not getting set. We need to check the value and try again if needed.
+      EPHEMERAL_CONTENT_ORIGIN=https://${HOST}
+      while true; do
```

Comment:
Could this get into some sort of infinity loop?

Reply:
yes